concourse/scripts/ssh.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -eu
+
+key=/tmp/concourse_id_rsa.$RANDOM
+trap "rm -f $key" SIGINT SIGTERM EXIT
```

Comment:
Are the `SIGINT` and `SIGTERM` necessary here?  I thought that an `EXIT` trap would always fire when the script exited regardless of the cause.

If they are necessary, we need to update the exit traps in several of our other scripts.